### PR TITLE
Add subchannels to funding wheel

### PR DESCRIPTION
**Pull request by NiloCK**
PR adds some niceties to the channel wheel:

1. "my" balance is rotated to bottom of wheel, for "filling a bucket" feel
2. locked funds are demarcated by channel (rather than mine, theirs)
  - with a visual indicator of internal balance being toward 'me' or 'them'
  - channels with more 'me' funds are displayed 'lowest', consistent with gravity effect in (1.)

Diff is not very easy to read here :/.

### CODE REVIEW

File: packages/site/src/components/NetworkBalance.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
-import { NetworkBalance, NetworkBalanceProps } from './NetworkBalance';
+import { NetworkBalance, NetworkBalanceProps, VirtualChannelBalanceProps } from './NetworkBalance';
 
 export default {
   title: 'NetworkBalance',
```
Comment (lalexgap) [inline packages/site/src/components/NetworkBalance.stories.tsx:6]: Kinda outside the scope of this PR but is it possible to have some kind of description field? I feel like it would make storybook easier to understand if we could describe the context for components.

File: packages/site/src/components/NetworkBalance.tsx [hunk 2]
```diff
@@ -94,40 +174,36 @@ export const NetworkBalance: React.FC<NetworkBalanceProps> = (props) => {
               })}
               radius={42}
               data={data}
-              label={({ dataEntry }) => prettyPrintWei(myBalanceFree)}
+              label={({ dataEntry }) => prettyPrintWei(myTotal)}
               labelPosition={0}
               segmentsStyle={(idx) => ({ color: 'red' })}
+              paddingAngle={data.length > 1 ? 0.5 : 0}
+              startAngle={90 - myBalanceFreePercentage * 1.8} // 1.8 = 180 degrees / 100 (percentage)
```
Comment (lalexgap) [inline packages/site/src/components/NetworkBalance.tsx]: Nit but I find this slightly hard to parse. I wonder if we should pull it out to it's own variable?

File: packages/site/src/components/NetworkBalance.tsx [hunk 3]
```diff
@@ -1,82 +1,162 @@
...
+  }
+
+  return toExtremes;
+}
+
 export const NetworkBalance: React.FC<NetworkBalanceProps> = (props) => {
   const {
     myBalanceFree,
-    myBalanceLocked,
+    // myBalanceLocked,
```
Comment (lalexgap) [inline packages/site/src/components/NetworkBalance.tsx]: Remove instead of comment out?

File: packages/site/src/components/NetworkBalance.tsx [hunk 4]
```diff
@@ -1,82 +1,162 @@
...
     theirBalanceFree,
-    theirBalanceLocked,
+    // theirBalanceLocked,
```
Comment (lalexgap) [inline packages/site/src/components/NetworkBalance.tsx]: Remove instead of comment out?

File: packages/site/src/components/NetworkBalance.tsx [hunk 5]
```diff
@@ -1,82 +1,162 @@
...
+    (acc, x) => acc + x.budget,
+    BigInt(0),
+  );
+
+  const total = myBalanceFree + theirBalanceFree + lockedTotal;
+  const myTotal =
+    myBalanceFree +
+    sortedVirtualChannels.reduce(
+      (acc, x) =>
+        acc + (x.budget * BigInt(Math.round(100 * x.myPercentage))) / 100n,
```
Comment (lalexgap) [inline packages/site/src/components/NetworkBalance.tsx]: I'm pretty rusty on all this stuff,  but does it matter that we're rounding here?

**Review by lalexgap — APPROVED** [review]

File: packages/site/src/components/NetworkBalance.tsx (hunk 5 above)
Comment (NiloCK) [reply packages/site/src/components/NetworkBalance.tsx]: Added some extra decimal points on the rounding here:

e717701

Given that this calculation only affects a UI widget, and the data (shouldn't) propogate further or influence any "business" decisions, I think some loss of accuracy here is OK.

File: packages/site/src/components/NetworkBalance.tsx (hunk 2 above)
Comment (NiloCK) [reply packages/site/src/components/NetworkBalance.tsx]: e717701

This value is definitely hard to parse when you're not actively fiddling with it. Even when you are!

Added an explanation and a two-step calculation.

File: packages/site/src/components/NetworkBalance.stories.tsx (hunk 1 above)
Comment (NiloCK) [reply packages/site/src/components/NetworkBalance.stories.tsx:6]: Not 100% sure whether or not this is possible in the way I understand your request. Can circle back to it later.
